sources/bot/storage/spreadsheet/tests/tests_spreadsheet_handler.py

```diff
@@ -1,13 +1,137 @@
+import json
+import os
+import sys
+from datetime import datetime
+
+import apiclient
+import httplib2
+from googleapiclient.errors import HttpError
+from oauth2client.service_account import ServiceAccountCredentials
+
 from .base_tests_spreadsheet_handler import BaseTestsSpreadsheetHandler
 
 
 class TestsSpreadsheetHandler(BaseTestsSpreadsheetHandler):
-    # todo: Implement tests handler
-    def __init__(self, spreadsheet_id: str, token: str):
-        pass
+    def __init__(self, credentials_file_name: str):
+        self._loaded_tests = {}
+        self._current_test_id = ""
+        self._http_auth = ServiceAccountCredentials.from_json_keyfile_name(
```

Comment:
We have a class that does this kind of logic. We were talking about it. Please, use it.